[All Hosts] (auth) add info for revoking permissions (#4549)
OfficeDev/office-js-docs-pr: docs/develop/grant-admin-consent-to-an-add-in.md

---
title: Grant or revoke administrator consent to the add-in
description: Learn how to grant or revoke administrator consent to your add-in.
ms.date: 05/06/2023
ms.localizationpriority: medium
---

# Grant or revoke administrator consent to the add-in

> The sections below show how to grant and revoke administrator consent for the permissions that you add-in needs to run.

Reviewer: ```suggestion
This article shows how to grant or revoke administrator consent for the permissions that your add-in needs to run.
```
